parlai/crowdsourcing/tasks/turn_annotations_static/analysis/compile_results.py

```diff
@@ -22,6 +22,10 @@ class TurnAnnotationsStaticResultsCompiler(AbstractTurnAnnotationResultsCompiler
 
     Change PROBLEM_BUCKETS in task_config/annotations_config.json to be the buckets that
     you are asking crowdsource workers to annotate with.
+
+    NOTE this script directly accesses Mephisto files rather than using the DataBrowser.
```

Comment:
good to flag - I have some outstanding that ports this over to DataBrowser, but some work will need to be done first to fully test it, convert the unit tests over, handle all old code that relies on this, etc.

Reply:
Of course - it's a bit of work for sure but I wanted to make sure that anyone stumbling across this in the meantime knows what's up.